conf/application.conf

```diff
@@ -188,3 +188,9 @@ microservice {
   }
```

Comment:
Tests for missing info are not possible in the frontend due to all attributes of the NSIUserInfo model in the frontend not being options.